cpp/src/slice2java/Gen.cpp

```diff
@@ -1474,22 +1446,17 @@ Slice::JavaVisitor::writeDispatch(Output& out, const InterfaceDefPtr& p)
 
             for (const auto& value : values)
             {
-                const string typeS =
+                const string ts =
```

Comment:
I would remove `ts` variable, and use typeToString directly. The variable is only used once and doesn't make the code any clear.